examples/specialty_plots/radar_chart.py

```diff
@@ -82,7 +82,7 @@ def _gen_axes_patch(self):
                 return RegularPolygon((0.5, 0.5), num_vars,
                                       radius=.5, edgecolor="k")
             else:
-                raise ValueError("unknown value for 'frame': %s" % frame)
```

Comment:
do you want to change these (and below) to check_in_list while you're at it? (optional, just suggesting...)

Reply:
Rather not. `cbook._check_in_list` is private, and we shouldn't use it in public examples.

Reply:
fair enough, but some below (e.g. axis.py) could use that

Reply:
Added `_check_in_list` in axis.py for the better exception message, but still left the `ValueError` in the else branch (will catch if the list of of accepted parameters and the code handling these go out of sync).

Reply:
Added `_check_in_list` in axis.py for the better exception message, but still left the `ValueError` in the else branch (will catch if the list of of accepted parameters and the code handling these go out of sync).